io.github.Soundux.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.Soundux
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: soundux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=host:ro
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
What is it needed for?

Reply:
I think this can be improved.
I added write access to home because it writes a config file. I also added read only access to root because maybe the user wants to add a folder that's under /run/media

Reply:
I mean `--talk-name=org.freedesktop.Flatpak` here.

Reply:
We use the output of `pacmd` to get information from PulseAudio and move source outputs

Reply:
Wouldn't be better to bundle `pacdm` into the app? Running commands on host is major sandbox hole.

Reply:
How can we achieve this? It is a command line utility from PulseAudio

Reply:
```diff
diff --git a/io.github.Soundux.yml b/io.github.Soundux.yml
index 6795f7d..bd36fb2 100644
--- a/io.github.Soundux.yml
+++ b/io.github.Soundux.yml
@@ -13,8 +13,19 @@ finish-args:
   - --filesystem=/media:ro
   - --filesystem=/run/media:ro
   - --filesystem=~/.config/Soundux
-  - --talk-name=org.freedesktop.Flatpak
 modules:
+  - name: pacmd
+    buildsystem: simple
+    build-commands:
+      - ./autogen.sh
+      - make -C src pacmd
+      - install -Dm644 src/.libs/libpulsecommon-14.0.so /app/lib/libpulsecommon-14.0.so
+      - install -Dm755 src/.libs/pacmd /app/bin/pacmd
+    sources:
+      - type: git
+        url: https://github.com/pulseaudio/pulseaudio
+        tag: v14.0
+
   - name: soundux
     buildsystem: cmake-ninja
     config-opts:
```

Attempts to run pacmd give me this though:

```
No PulseAudio daemon running, or not running as session daemon.
```

So I guess that's not the way. Maybe something in pavucontrol source code would give you inspiration? It's available on Flathub: https://github.com/flathub/org.pulseaudio.pavucontrol

Reply:
I think it's somehow possible with the `PULSE_SERVER` environment variable but I don't know what to set it to.
What I find weird is that `pactl` works just fine. (f.e. `pactl list source-outputs`). Maybe we should just switch to that

Reply:
`pactl list source-outputs` does not return anything for me (neither in flatpak or on the host) but if you tested it and it works properly, then yeah, makes sense to me.

Reply:
It only returns something when you have an application open which listens to your microphone, e.g. Discord or TeamSpeak
`pactl list sources` should always return something

Reply:
We adjusted it to use `pactl` instead of `pacmd` and it works now. I have removed the permission.